cpp/include/cudf/aggregation.hpp

```diff
@@ -488,5 +490,39 @@ std::unique_ptr<Base> make_merge_sets_aggregation(null_equality nulls_equal = nu
 template <typename Base = aggregation>
 std::unique_ptr<Base> make_merge_m2_aggregation();
 
+/**
+ * @brief Factory to create a TDIGEST aggregation
+ *
+ * Produces a tdigest (https://arxiv.org/pdf/1902.04023.pdf) column from input values.
+ * The input aggregation values are expected to be fixed-width numeric types.
+ *
+ * @param delta Parameter controlling compression level and accuracy on subsequent
+ * queries on the output tdigest data.  Delta places an upper bound on the size of
+ * the computed tdigests: A delta of 1000 will result in a tdigest containing no
+ * more than 1000 centroids (32 bytes each). Higher values of delta result in more
+ * accurate tdigest information.
+ *
+ * @returns A TDIGEST aggregation object.
+ */
+template <typename Base = aggregation>
+std::unique_ptr<Base> make_tdigest_aggregation(int delta = 1000);
+
+/**
+ * @brief Factory to create a MERGE_TDIGEST aggregation
+ *
+ * Produces a tdigest (https://arxiv.org/pdf/1902.04023.pdf) column from input values.
+ * The input aggregation values are be tdigests produced by the TDIGEST aggregation.
```

Comment:
Do the `delta` values need to be exactly equal for all tdigests being merged? 
```suggestion
 * Merges the results from a previous aggregation resulting from a `make_tdigest_aggregation` or `make_merge_tdigest_aggregation` to produce a new a tdigest (https://arxiv.org/pdf/1902.04023.pdf) column. 
```

Reply:
No.  The delta values can be completely independent.